### ISSUE TITLE
Error in vagrant up site (provisioning)

### ISSUE THREAD

**Issue by fgregg** [opencivicdata/vagrant-opencivicdata.org#8]
```bash 
> vagrant up site
> vagrant provision
...
Downloading/unpacking static3 (from dj-static->-r /projects/opencivicdata.org/src/opencivicdata.org/requirements.txt (line 9))
  Downloading static3-0.6.0.tar.gz
  Running setup.py (path:/projects/opencivicdata.org/virt/build/static3/setup.py) egg_info for package static3
    Traceback (most recent call last):
      File "<string>", line 17, in <module>
      File "/projects/opencivicdata.org/virt/build/static3/setup.py", line 52, in <module>
        long_description=open('README.rst').read(),
      File "/projects/opencivicdata.org/virt/lib/python3.4/encodings/ascii.py", line 26, in decode
        return codecs.ascii_decode(input, self.errors)[0]
    UnicodeDecodeError: 'ascii' codec can't decode byte 0xc2 in position 2097: ordinal not in range(128)
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):

  File "<string>", line 17, in <module>

  File "/projects/opencivicdata.org/virt/build/static3/setup.py", line 52, in <module>

    long_description=open('README.rst').read(),

  File "/projects/opencivicdata.org/virt/lib/python3.4/encodings/ascii.py", line 26, in decode

    return codecs.ascii_decode(input, self.errors)[0]

UnicodeDecodeError: 'ascii' codec can't decode byte 0xc2 in position 2097: ordinal not in range(128)

----------------------------------------
Cleaning up...
Command python setup.py egg_info failed with error code 1 in /projects/opencivicdata.org/virt/build/static3
Storing debug log for failure in /projects/opencivicdata.org/.pip/pip.log


FATAL: all hosts have already failed -- aborting
```

**Comment by jamesturk** [comment]
looks like a bug in static3 0.6.0, they just released 0.6.1 which I presume fixes this, try again?

**Comment by fgregg** [comment]
yup, works now.
